packages/blueprints/.eslintignore

```diff
@@ -9,5 +9,4 @@
 # misc
 !.*
 .*/
-/src/blueprints/
```

Comment:
The package happens to have a folder called `src/blueprints`, so this line stops Prettier's auto-fix from working.